src/Akeneo/Platform/Bundle/AnalyticsBundle/Query/Sql/ApiConnectionCount.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Platform\Bundle\AnalyticsBundle\Query\Sql;
+
+use Akeneo\Connectivity\Connection\Domain\Settings\Model\ValueObject\FlowType;
+use Akeneo\Tool\Component\Analytics\ApiConnectionCountQuery;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\FetchMode;
+
+/**
+ * @author Pierre Jolly <pierre.jolly@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ApiConnectionCount implements ApiConnectionCountQuery
+{
+    /** @var Connection */
+    private $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public function fetch(): array
+    {
+        $query = <<<SQL
+SELECT flow_type, count(flow_type) AS "flow_type_count", auditable
+FROM akeneo_connectivity_connection
+GROUP BY flow_type, auditable
+SQL;
+
+        $data = $this->connection->executeQuery($query)->fetchAll(FetchMode::ASSOCIATIVE);
+
+        return $this->normalize($data);
+    }
+
+    private function normalize(array $data): array
+    {
+        $normalized[FlowType::DATA_SOURCE] = ['tracked' => 0, 'untracked' => 0];
+        $normalized[FlowType::DATA_DESTINATION] = ['tracked' => 0, 'untracked' => 0];
+        $normalized[FlowType::OTHER] = ['tracked' => 0, 'untracked' => 0];
+
+        foreach ($data as $row) {
+            $status = $row['auditable'] ? 'tracked':'untracked';
```

Comment:
```suggestion
            $status = (bool) $row['auditable'] ? 'tracked':'untracked';
```

As any data returned by PDO is a string, it's a little bit better for readability: it says you explicitely expects a boolean.